StortSpelProjekt/Engine/src/Renderer/GPUMemory/ConstantBufferView.cpp

```diff
@@ -0,0 +1,42 @@
+#include "stdafx.h"
+#include "ConstantBufferView.h"
+
+#include "../DescriptorHeap.h"
+#include "Resource.h"
+
+ConstantBufferView::ConstantBufferView(
+	ID3D12Device5* device,
+	DescriptorHeap* descriptorHeap_CBV_UAV_SRV,
```

Comment:
Don't see reason to put in the dHeap in the constructor. You are putting it in createConstantBufferView

Reply:
I disagree and this is is a good solution